akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

```diff
@@ -45,6 +45,25 @@ Use plain `system.actorOf` instead of the DSL to create Actors if you have been
 
 `actorFor` has been deprecated since `2.2`. Use `ActorSelection` instead.
 
+### Untyped actor removed
+
+`UntypedActor` has been depcated since `2.5.0`. Use AbstractActor instead.
+
+### UntypedPersistentActor removed
+
+Use `AbstractPersistentActor` instead.
+
+### UntypedPersistentActorWithAtLeastOnceDelivery removed
+
+Use AbstractPersistentActorWithAtLeastOnceDelivery instead.
```

Comment:
Back ticks for all types in .md files or for none - consistency. Most seem to have it so let's keep with convention.

Reply:
or `@apidoc`